README.md

```diff
@@ -1,51 +1,83 @@
+
+
+### Team Member Data 
+
+Data for each team member is represented in a JSON object. Each data object should contain the following properties. 
+
+| Property | Type | Status | Description |
+| ------- | ------- | ------- | ------- |
+| `firstName` | `string` | **required** | team member's first name |
+| `lastName` | `string` | **required** | team member's last name |
+| `suffix` | `string` | optional | team member's name suffix, if they have one |
+| `nickname` | `string` | unused | team members's nickname, if they have one |
+| `department` | `string` | unused | |
+| `jobTitle` | `string` | **required** | team member's job tile |
+| `role` | `"Developer" \| "UX"  \| "FED" \| "Designer" \| "PM" \| ""` | **required** | used to assign a team member to a matching filter. **CAUTION** Adding additional roles will affect design in potentially unstable ways |
+| `techLead` | `boolean` | unused | is team member able to serve as a tech lead |
+| `skillLevel` | `number` | unused | |
+| `active` | `boolean` | unused | |
+| `cohortLeader` | `string` | optional | cohort team member is leader of, if they lead one | 
+| `cohortParticipant` | `string` | optional | cohort team member is participant in |
+| `internalResponsibility` | `string[] ` | unused | |
+| `discoveryLead` | `boolean` | unused | |
+| `certifications` | `string[]` | optional | team members's certifications, if they have any. `string[]` can be optionally substituted with a `string` of comma separated values *ex* `"PMP, PSM"` |
+| `bio` | `string` | optional | team member's bio |
+| `funFacts` | `string[]` | **required** | fun facts about the team member. can be [] if none |
+| `technologies` | `string[]` | optional | technologies a team member is proficient in.`string[]` can be optionally substituted with a `string` of comma separated values *ex* `"JavaScript, HTML, CSS` |
+| `expertise` | `string[]` | optional | technologies or skill sets a team member has expertise in. `string[]` can be optionally substituted with a `string` of comma separated values *ex* `"JavaScript, HTML, CSS` |
+| `strengths` | `string[]` | optional | team member's strengths. `string[]` can be optionally substituted with a `string` of comma separated values *ex* `"JavaScript, HTML, CSS` |
+| `interests` | `string[]` | optional | team member's interests. `string[]` can be optionally substituted with a `string` of comma separated values *ex* `"JavaScript, HTML, CSS` |
+| `currentProjects` | `project[]` | optional | current projects the team member is on. [See Below for project data specification](#project-data) |
+| `pastProjects`| `string[]` | **required** | past projects a team member has worked on |
+| `photo` | `string` | **required** | url of team member's photo |
+| `weeklyCapacity` | `number` | **required** | weekly capacity of maximum billable hours for a team member |
+| `forecastedHours` | `number` | **required** | forecasted weekly billable hours for a team member |
+| `pto` | `number` | unused | |
```

Comment:
I think this will be super useful for defining the final API contract! Calling out the things that are unused might even help trim it down in the end.